### ISSUE TITLE
Warnings about non supported, non global ARIA attributes use on ARIA roles

### ISSUE THREAD

**Issue by spectranaut** [validator/validator#1716]
ARIA is dropping support of `aria-expanded` on role `listbox`: https://github.com/w3c/aria/pull/1862

This means the following HTML is no longer valid:
```
<div role="listbox" aria-label="foo" aria-expanded="true">
</div>
```

I noticed, however, that vnu does not raised any errors when a non-supported, non global ARIA attribute is used on a role. For example, the following does not raise an error:
```
<div role="heading" aria-rowspan="3" id="oops" aria-expanded="true">
</div>

```

**Comment by pkra** [comment]
:+1: for https://github.com/validator/validator/issues/1930

**Comment by sideshowbarker** [comment]
> when a non-supported, non global ARIA attribute is used on a role

Are there test cases somewhere for that?

**Comment by sideshowbarker** [comment]
I guess this is about the requirements at https://www.w3.org/WAI/standards-guidelines/act/rules/5c01ea/proposed/?

If so, I see that has tests it links to there — so I guess I can start testing with those.

**Comment by sideshowbarker** [comment]
> I guess this is about the requirements at [w3.org/WAI/standards-guidelines/act/rules/5c01ea/proposed](https://www.w3.org/WAI/standards-guidelines/act/rules/5c01ea/proposed/)?
> 
> If so, I see that has tests it links to there — so I guess I can start testing with those.

Update: We’re already behaving as expected for all the cases which are marked _“Failed”_ there.

**Comment by sideshowbarker** [comment]
> ARIA is dropping support of `aria-expanded` on role `listbox`: [w3c/aria#1862](https://github.com/w3c/aria/pull/1862)
> 
> This means the following HTML is no longer valid:
> 
> ```
> <div role="listbox" aria-label="foo" aria-expanded="true">
> </div>
> ```

Fixed in https://github.com/validator/validator/commit/f6c2fe281d and deployed:

https://validator.w3.org/nu/?showsource=yes&doc=data%3Atext%2Fhtml%3Bcharset%3Dutf-8%3Bbase64%2CPCFkb2N0eXBlIGh0bWw%2BPGh0bWwgbGFuZz48bWV0YSBjaGFyc2V0PXV0Zi04Pg0KPHRpdGxlPnRlc3Q8L3RpdGxlPg0KPGRpdiByb2xlPSJsaXN0Ym94IiBhcmlhLWxhYmVsPSJmb28iIGFyaWEtZXhwYW5kZWQ9InRydWUiPjwvZGl2Pg%3D%3D

> For example, the following does not raise an error:
> 
> ```
> <div role="heading" aria-rowspan="3" id="oops" aria-expanded="true">
> </div>
> ```

For that, it seems we already had the checker reporting the following errors:

- _Error: Attribute `aria-rowspan` not allowed on element [`div`](https://html.spec.whatwg.org/multipage/#the-div-element) at this point._
- _Error: Element [`div`](https://html.spec.whatwg.org/multipage/#the-div-element) is missing required attribute aria-level._
 
> I noticed, however, that vnu does not raised any errors when a non-supported, non global ARIA attribute is used on a role.

If there are remaining cases where the checker’s still not conforming to particular requirements around that, those could be addressed in follow-up issues.
